backend/repositories/messageRepository.js

```diff
@@ -3,12 +3,57 @@ var Message = require('../schemas/messageSchema');
 
 MessageRepository.prototype = new Repository();
 
-MessageRepository.prototype.some = some;
-
+MessageRepository.prototype.createMessage = createMessage;
+MessageRepository.prototype.getMessage = getMessage;
+MessageRepository.prototype.updateMessage = updateMessage;
+MessageRepository.prototype.deleteMessage = deleteMessage;
 
 function MessageRepository() {
 	Repository.prototype.constructor.call(this);
 	this.model = Message;
 };
 
+function createMessage(messageTitle, messageDescription, messageAuthor, messageProject, messageComments, messageFiles, callback){
+    var model = this.model;
+    var newMessage = new model({
+        title: messageTitle,
+        description: messageDescription,
+        author: messageAuthor,
+        project: messageProject,
+        date: Date.now(),
+        comments: messageComments,
+        files: messageFiles
+    });
+	newMessage.save(callback);
+};
+
+function getMessage(messageId, callback){
+    var model = this.model;
+    var query = model.find({_id: messageId});

```

Comment:
when you search by id - it is better to use findOne method to get object instead of array with one object